bvm/ballerina-core/src/main/java/org/ballerinalang/util/metrics/Snapshot.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.util.metrics;
+
+import java.time.Duration;
+import java.util.Arrays;
+
+/**
+ * Snapshot of all distribution statistics at a point in time.
```

Comment:
Shall we add @ Since version for all new classes

Reply:
Added with revision - 17638a71c0c7535d68f2dd262a60424d75524c8c.